Project.toml

```diff
@@ -21,10 +21,10 @@ RecipesBase = "3cdcf5f2-1ef4-517c-9805-6587b60abb01"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
 [compat]
-BioAlignments = "2"
+BioAlignments = "2, 3"
```

Comment:
Drop the `2` in each case.

Reply:
BioCore is a unique one. I think it is still v2 actually. Hasn't been updated in a very long time (5 years)